### PR TITLE
ci(workflow): add default needs repro text

### PR DESCRIPTION
**Pull request by danielleroux**
## 🆕 What is the new behavior?

Add label automation

### CODE REVIEW

File: .github/workflows/issue-needs-repo.yml
Comment (nuke-ellington) [inline .github/workflows/issue-needs-repo.yml:1]: Better filename issue-needs-repro.yml?

File: .github/workflows/issue-needs-repo.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+on:
+  issues:
+    types: [labeled]
+
+jobs:
+  comment-issue:
+    if: ${{ github.repository == 'siemens/ix' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Need reproduction comment created
```
Comment (nuke-ellington) [inline .github/workflows/issue-needs-repo.yml:12]:
```suggestion
      - name: Needs reproduction comment created
```

File: .github/workflows/issue-needs-repo.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        if: "contains(github.event.label.name, 'needs reproduction')"
+        uses: actions-cool/issues-helper@v3
+        with:
+          actions: 'create-comment'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          body: |
+            Thanks for the issue! This issue has been labeled as `needs reproduction`.
```
Comment (nuke-ellington) [inline .github/workflows/issue-needs-repo.yml:19]:
```suggestion
            Thank you for creating this issue! It has been labeled as `needs reproduction`.
```

**Review by nuke-ellington — CHANGES_REQUESTED** [review]

File: .github/workflows/issue-needs-repo.yml
Comment (danielleroux) [reply .github/workflows/issue-needs-repo.yml:1]: No

**Review by nuke-ellington — APPROVED** [review]

**Pull request merged**